python/tests/test_fft.py

```diff
@@ -85,7 +77,33 @@ def test_fftn(self):
                 x = a
                 if op in ["rfft2", "rfftn"]:
                     x = r
-                self.check_mx_np(op, x, axes=ax, s=s)
+                mx_op = getattr(mx.fft, op)
+                np_op = getattr(np.fft, op)
+                self.check_mx_np(mx_op, np_op, x, axes=ax, s=s)
+
+    @unittest.skipIf(not mx.metal.is_available(), "Metal is not available")
+    def test_fft_gpu(self):
+        shape = (16, 4, 8)
+        # np.fft.fft always uses double precision complex128
+        # mx.fft.fft only supports single precision complex64
+        # hence the fairly tolerant equality checks.
+        atol = 1e-4
+        rtol = 1e-4
+        np.random.seed(7)
+        with mx.stream(mx.gpu):
```

Comment:
I would just run these tests on both devices rather than skipping them for the CPU and specifying the context manager. The goal is to remove the context manager for the CPU above once all the ops can run on the GPU.

Reply:
(usually we don't specify the device in the tests but just run the full test suite twice with the CPU and GPU as default).